packages/portalnetwork/src/subprotocols/beacon/types.ts

```diff
@@ -5,6 +5,8 @@ import { Bytes32Type } from '../types.js'
 export const MainnetGenesisValidatorsRoot =
   '0x4b363db94e286120d76eb905340fdd4e54bfe9f06bf33ff6cf5ad27f511bfe95'
 
+export const MainnetGenesisTime = 1606824023
```

Comment:
you can import this data from the lodestar :
```
 import {networksChainConfig} from "@lodestar/config/networks";
 ```
 networksChainConfig has genesis root and time for all major networks (and gets updated as well)